android-base/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Sun Jan 19 20:22:02 JST 2020
```

Comment:
Thank you! 👍 
This code is for the root directory. So we don't need this file. 🙇

Reply:
delete it